src-tauri/src/farmer.rs

```diff
@@ -41,16 +39,16 @@ impl Default for ArchivingFrom {
 }
 
 #[tauri::command]
-pub(crate) async fn farming(path: String, reward_address: String, plot_size: u64) -> bool {
-    // create a channel to listen for farmer errors, and restart another farmer instance in case any error
-    let (error_sender, mut error_receiver): (Sender<()>, Receiver<()>) = channel(1);
-
+pub(crate) async fn farming(
```

Comment:
Let's start [adding documentation](https://github.com/subspace/subspace-desktop/issues/297) with our changes now. This will make it easier for us to ramp up to fuller coverage. :pray:

Reply:
I am already adding it in another branch - `add-code-doc`

Reply:
In Rust it is common practice (typically enforced by lints) to require documenting all public API surface. Internal API (like this) is typically documented only if it is not trivial or requires it for some other reason. Otherwise might be just a waste of time commenting obvious things for the sake of doing it.

Reply:
I agree with Nazar on this one. 
The exception maybe this: the functions that are annotated with `#[tauri::command]` are exposed to frontend, so they are not staying exactly in the crate, although they are `pub(crate)`. 
If you want, I can document all the `#[tauri::command]` ones, because they are somewhat like `public API` to some degree, but not completely...

Decision is yours :)